### ISSUE TITLE
client on linux forcing to use the keyring

### ISSUE THREAD

**Issue by Codeberg-AsGithubAlternative-buhtz** [owncloud/client#7014]
I am using ownCloud client 2.4.1+dfsg-1~bpo9+1 on Debian stable with XFCE desktop environment.
Please see this detailed version information.
```
$ apt-cache policy owncloud-client
owncloud-client:
  Installiert:           2.4.1+dfsg-1~bpo9+1
  Installationskandidat: 2.4.1+dfsg-1~bpo9+1
  Versionstabelle:
 *** 2.4.1+dfsg-1~bpo9+1 100
        100 http://ftp.debian.org/debian stretch-backports/main amd64 Packages
        100 /var/lib/dpkg/status
     2.2.4+dfsg-2 500
        500 http://ftp.tu-chemnitz.de/pub/linux/debian/debian stretch/main amd64 Packages
        500 http://deb.debian.org/debian stretch/main amd64 Packages

```

It loosk like that ownCloud clients tries to store the password in the keyring. I don't want that but I can not see any option in ownCloud to deactivate that behaviour.

I tried a workaround with modifying the access rights and ownership of the keyring file like this
`-rw------- 1 root root 0 Feb  4 15:31 Vorgabe.keyring`
In that case ownCloud is unable to login. I type my password again and again but ownCloud doesn't accept it anymore.

**Comment by ckamm** [comment]
@noMICROSOFTbuhtz Indeed, there's currently no way of disabling the use of the keychain.

If someone wanted to implement the feature, there'd be a few options. The client uses QtKeychain fo access the keychain and links to it dynamically. It could be improved to have a no-backend mode that just makes read/write attempts come back with errors.

Another option would be to start in owncloud-client code and adjust all places where the QKeychain namespace and the keychain.h include is used.

**Comment by pascal06** [comment]
Hello,

I'm just an user but indeed, I think this behaviour is very annoying. These days, I decided to use hardware key (Yubikey) to login into my computer. So yesterday I changed my original password (that was 3 years old !) to a shorter one plus the Yubikey. This morning, after having rebooted my computer, the "keyring" says my password has changed and asked me the old one. So I guess that each time I will change my system password keyring will ask my first password that I used 3, 4, maybe 10 years ago... It's a non-sense for me, sorry.
It could be great I don't know how as I'm not a specialist to remove this behaviour. Thanks a lot in advance.

Pascal

```
$ apt-cache policy owncloud-client
owncloud-client:
  Installé : 2.4.1+dfsg-1
  Candidat : 2.4.1+dfsg-1
 Table de version :
 *** 2.4.1+dfsg-1 500
        500 http://fr.archive.ubuntu.com/ubuntu bionic/universe amd64 Packages
        100 /var/lib/dpkg/status
```
```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.6 LTS
Release:	18.04
Codename:	bionic
```
